metrics/tests/test_different_workdir.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import os, json, pytest, subprocess
+from tempfile import NamedTemporaryFile
+
+
+# path of test data
+cur_dir = os.path.dirname(os.path.abspath(__file__))
+file_path = cur_dir + "/../eval.py"
+video_y4m_path = cur_dir + "/data/test.y4m"
+video_yuv_path = cur_dir + "/data/test.yuv"
+audio_path = cur_dir + "/data/test.wav"
+dst_network_log = cur_dir + "/data/alphartc.log"
+workdir = cur_dir + "/data"
+
+
+def run_and_check_result(cmd):
+    cmd_result = subprocess.run(cmd, check=True, stdout=subprocess.PIPE, stderr=subprocess.STDOUT, encoding="utf8", cwd=workdir)
+    
+    data = json.loads(cmd_result.stdout)
+    assert "video" in data
+    assert type(data["video"]) == float
+    assert data["video"] >= 0 and data["video"] <= 100
+    assert "audio" in data
+    assert type(data["audio"]) == float
+    assert data["audio"] >= 0 and data["audio"] <= 100
+    assert "network" in data
+    assert type(data["network"]) == float
+    assert data["network"] >= 0 and data["network"] <= 100
```

Comment:
and other places...

Reply:
you can select the whole class and ask copilot to review it and give you some suggestions to improve it, and it will generate code which you can choose to replace yours or not based on the code quality.